packages/frontend/src/page/hooks/useWatchKPITokenData.ts

```diff
@@ -0,0 +1,46 @@
+import { KPI_TOKEN_ABI } from "@carrot-kpi/sdk";
+import { useEffect, useState } from "react";
+import { type Address, useContractReads } from "wagmi";
+
+interface WatchKPITokenDataParams {
+    kpiTokenAddress?: Address;
+}
+
+interface KPITokenData {
+    finalized: boolean;
+    data: Address;
```

Comment:
It should be the same, but could this be `Hex` instead of `Address`? It makes more sense semantically.

Reply:
Fixed in 269b3e4827e7970de94855a1c931891edfe13c21.